tox.ini

```diff
@@ -1,14 +1,15 @@
 [tox]
 envlist =
     py{3.6,3.7,3.8}-django-{22,30,31,master}
+isolated_build = True
 
 [testenv]
-commands = ./test.sh {posargs}
+commands =
```

Comment:
What about adding poetry in the allowlist instead of in the list of deps? [source](https://github.com/python-poetry/poetry/blob/02d4d0c4fff8270c01fde399a7fb6ad1ad51f56c/docs/docs/faq.md#is-tox-supported), [source2](https://tox.readthedocs.io/en/latest/config.html#conf-allowlist_externals)

```
allowlist_externals = poetry
```

Reply:
per comment https://github.com/aldjemy/aldjemy/pull/109#discussion_r519961161
I really don't know what's best.

Reply:
I think either way is fine. But using tox to manage poetry might lead to poetry installation issues. I'm not familiar with what Python versions aldjemy supports but Poetry for example only officially supports versions 3.5 and up though we are in the process of deprecating 3.5 soon.

Reply:
IMO, tox, if used, should manage the dependency installer. If you install dependencies, then run tox, it will install the dependencies again, because tox is managing virtual environments.